tower-mock/src/lib.rs

```diff
@@ -336,6 +337,32 @@ impl<T, E> Future for ResponseFuture<T, E> {
     }
 }
 
+// ===== impl Error =====
+
+impl<T> fmt::Display for Error<T>
```

Comment:
This change is just to get things to work for now. It will be changed once `tower-mock` is updated to use `Box<Error>`.